### PR TITLE
Feature/petitions page 3

### PR DESCRIPTION
**Pull request by mtauer**
@judithhartmann To fix merge conflicts I had change the `DefaultContentBlock` a bit as I introduced layout props for the title and the content. Can you please check if the default content block still works on the other pages as expected?

### CODE REVIEW

**Comment by judithhartmann** [comment]
The effect that I wanted to achieve with the `InnerContainer` is now overwritten: 

image

The background color should be applied to the whole white area in between the carousels on the home page

**Comment by mtauer** [comment]
> The effect that I wanted to achieve with the `InnerContainer` is now overwritten:
> 
> image
> 
> The background color should be applied to the whole white area in between the carousels on the home page

Ok, will fix before this PR can be merged.

**Comment by mtauer** [comment]
Changes are in. There is now a `LayoutRow` component that defines the horizontal spacing for the components.
@judithhartmann Can you please check if all the pages still look as expected? (I could not find issues anymore)

File: src/components/contentBlocks/petitionTopic.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+const PetitionTopic: FC<PetitionTopicProps> = ({ contentComponent, title, titleBackgroundColor = colors.Grey400 }) => {
+  const HEADER_MAX_LINES = 5;
+  const repeatedTitle = _.repeat(`${title} `, 10);
+  const titleLineOffsets = useMemo(() => _.range(HEADER_MAX_LINES).map(() => _.random(0, 500)), []);
+  return (
+    <Container>
+      <HeroContainer backgroundColor={titleBackgroundColor}>
+        {_.range(HEADER_MAX_LINES).map((i:number) => (
+          <HeroTitleLine
+            aria-hidden
```
Comment (judithhartmann) [inline src/components/contentBlocks/petitionTopic.tsx]: 👍

**Pull request merged**